Dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM registry.redhat.io/3scale-amp2/system-rhel7:3scale2.14
+# FROM registry.redhat.io/3scale-amp2/system-rhel8:3scale2.15
+FROM registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-system-rhel8:1.17.0-529
```

Comment:
we should probably use a public image uri?

Reply:
You're right, but as 2.15 tag was not available, I was using this one locally and forgot to switch back before pushing :facepalm: